docs/troubleshooting.md

```diff
@@ -1,3 +1,4 @@
+## Sequencing Output
```

Comment:
Blank lines after all headings in this file

Reply:
Why? Those headers are old and seem to be rendering properly: https://miso-lims.readthedocs.io/projects/runscanner/en/latest/troubleshooting/